### ISSUE TITLE
Enable OpenMetrics in Datadog

### ISSUE THREAD

**Issue by RachalCassity** [department-of-veterans-affairs/va.gov-team#56736]
## Description

Statsd Metrics are not reported the same in Datadog than Prometheus. In the metrics, all "_" are replaced with "." OpenMetrics should be enabled in the DatadogAgents.

## Task

- [ ] TBD

## Acceptance Criteria

- [ ] Metrics being reported correctly

**Comment by oseasmoran73** [comment]
There are a couple of threads that talks more in detail about it. They can be found here [slack1](https://dsva.slack.com/archives/C04M7T1CZAS/p1681302040135869) [slack2](https://dsva.slack.com/archives/C04M7T1CZAS/p1681153881257159?thread_ts=1680792328.300899&cid=C04M7T1CZAS)

**Comment by oseasmoran73** [comment]
I have not had success in getting open metrics working for vets-api in EKS. I tried to follow template similar to [BRD](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/build/roles/datadog/vars/vets-api.yml#L2-L17) settings. I found a couple of cross references in the app-manifest. This is only applied in dev and based off contact with Kyle S & Trevor B, they confirmed its still not gathering metric

Trevor created a patch [PR](https://github.com/department-of-veterans-affairs/vets-api/pull/12392) from vets-api side to correct them, but it would be better to have it fixed from yaml
